src/lib/components/forms/ConfirmDelete.svelte

```diff
@@ -63,7 +66,7 @@ Confirm deletion or one or more documents.
     {/if}
 
     <Flex>
-      <Button type="submit" mode="danger" disabled={count === 0}>
+      <Button type="submit" mode="danger" disabled={count === 0 || !$me}>
```

Comment:
May also be a good idea to check for `!$me` inside the `onSubmit` handler, and bail if the user isn't signed in, in case someone edits the button to remove the disabled attribute.

Reply:
I'm pretty sure the form will just fail, because the cookie won't be there, but can add it anyway.